docs/source/format/Columnar.rst

```diff
@@ -76,7 +76,7 @@ concepts, here is a small glossary to help disambiguate.
 * **Nested type**: a data type whose full structure depends on one or
   more other child types. Two fully-specified nested types are equal
   if and only if their child types are equal. For example, ``List<U>``
-  is distinct from ``List<V>`` iff U and V are different types.
+  is distinct from ``List<V>`` if U and V are different types.
```

Comment:
"iff" is not a typo.
"iff" means "if and only if".

How about changing "iff" to "if and only if" not "if" like the previous sentence?

```suggestion
  is distinct from ``List<V>`` if and only if U and V are different types.
```

Reply:
If we include this change, we should change the PR title (this is not a typo).
If we don't include this change, we can keep the current PR title.

Reply:
Hi, I think `if and noly if` seems a bit verbose because it appears in the line above.

Reply:
I think that we should choose correctness than simpleness here because we explain specification here.
If we can keep this "iff", I can accept and merge this. If we can't, we need other commiter's review.

Reply:
@kou Well, I give up the modification for `iff` now.